modules/lsm/lsm_module_iface.c

```diff
@@ -166,6 +177,44 @@ _drive_update (UDisksObject   *object,
     }
 }
 
+static gboolean
+_lsm_local_check (UDisksObject *object)
+{
+  /* LsmLocalDisk interface will be available on all disk drive as
+   * we don't have reliable way to tell whether LED control works or not
+   * forehead, user have to try and handle errors.
```

Comment:
This is quite unfortunate. But I guess the HW vendors don't allow us do any better, right?